### PR TITLE
INS-2008: RequestsExecutor component

### PR DESCRIPTION
**Pull request by ruz**
it deals with artifacts and message bus, feeds to LogicExecutor

### CODE REVIEW

File: logicrunner/logicrunner_unit_test.go [hunk 1]
```diff
@@ -1417,19 +1253,20 @@ func (s *LogicRunnerTestSuite) TestImmutableOrder() {
...
 					panic("timeout on waiting for immutable request 2 pinged us")
-					return make([]byte, 0), nil, nil
+					return &reply.CallMethod{Result: []byte{1,2,3}}, nil
 				}
 			} else {
 				log.Info("notify immutable chan and exit")
 				immutableChan <- struct{}{}
 			}
 		} else {
+			fmt.Printf("WTF: %#v\n\n", transcript )
```
Comment (bigbes) [inline logicrunner/logicrunner_unit_test.go]: ??

File: logicrunner/requestsexecutor_test.go [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+
+func TestRequestsExecutor_Execute(t *testing.T) {
+	ctx := inslogger.TestContext(t)
+	mc := minimock.NewController(t)
+	defer mc.Finish()
+	defer mc.Wait(time.Minute)
+
+	//requestRef := gen.Reference()
+	//baseRef := gen.Reference()
+	//protoRef := gen.Reference()
```
Comment (aalekseev-ru) [inline logicrunner/requestsexecutor_test.go]: remove?

File: logicrunner/requestsexecutor_test.go [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+	mc := minimock.NewController(t)
+	defer mc.Finish()
+	defer mc.Wait(time.Minute)
+
+	requestRef := gen.Reference()
+	nodeRef := gen.Reference()
+	//protoRef := gen.Reference()
+	//objRef := gen.Reference()
+	//
+	//resId := gen.ID()
```
Comment (aalekseev-ru) [inline logicrunner/requestsexecutor_test.go]: and here?

Comment (ruz) [reply logicrunner/requestsexecutor_test.go]: done

File: logicrunner/requestsexecutor_test.go (hunk 2 above)
Comment (ruz) [reply logicrunner/requestsexecutor_test.go]: done

File: logicrunner/logicrunner_unit_test.go (hunk 1 above)
Comment (ruz) [reply logicrunner/logicrunner_unit_test.go]: removed

**Pull request merged**